tests/resources/policy_specification.json

```diff
@@ -0,0 +1,78 @@
+{
+  "name": "",
+  "owners": [
+    "amoo"
+  ],
+  "users": [
+    "rvela",
+    "marco"
+  ],
+  "userAccess": "some data here",
+  "approvers": [
+    "ricardo",
+    "luis"
+  ],
```

Comment:
Let's remove `owners`, `users`, `userAccess`, and `approvers` here (okay to leave them in the actual python code, class definitions, etc.)

Reply:
done